assets/packaging/app.spec.tmpl

```diff
@@ -1,6 +1,6 @@
 Name: {{.strippedProjectName}}
```

Comment:
nit: Can we move the files in `assets/packaging` into separate folders named after the packaging method they are intended for? e.g. `assets/packaging/linux-deb`, `assets/packaging/darwin-pkg`, etc.